lib/fetch-requests/meal-fetch-requests.js

```diff
@@ -1,15 +1,46 @@
 const $ = require('jquery')
 const url = "https://rocky-earth-59921.herokuapp.com";
 import {handleResponse} from '../response-handlers/initial-response-handler'
-import {renderMeals} from '../response-handlers/meal-response-handlers'
+import {renderMeals, renderFoods, reRenderMeal} from '../response-handlers/meal-response-handlers'
 
-function getAllMeals() {
+export function getAllMeals() {
   fetch(`${url}/api/v1/meals`)
     .then(response => handleResponse(response))
     .then(meals => renderMeals(meals))
     .catch(error => console.error({ error }))
 }
 
-module.exports = {
-  getAllMeals,
+export function getAllFoodsMealsPage() {
+  fetch(`${url}/api/v1/foods`)
+    .then(response => handleResponse(response))
+    .then(foods => renderFoods(foods))
+    .catch(error => console.error({ error }))
+}
+
+export function postFoodToMeals(mealId, foodId) {
+  var postOptions = {
+    method: "POST",
+    headers: {'Content-Type': 'application/json'},
+  }
+  fetch(`${url}/api/v1/meals/${mealId}/foods/${foodId}`, postOptions)
+  .then(response => handleResponse(response))
+  .then(getFoodsForOneMeal(mealId))
```

Comment:
Per our conversation this hopefully could be fixed by a response that should be waited on but not really used.
```response => getFoodsForOneMeal(mealId)```